AirCasting/SessionViews/SessionCartView.swift

```diff
@@ -107,7 +126,13 @@ private extension SessionCartView {
     }
     
     func displayButtons(threshold: SensorThreshold) -> some View {
-        HStack(spacing: 20){
+        HStack(spacing: 20) {
+            if isFollowing {
+                unFollowButton
+            } else if session.type == .fixed, !isFollowing {
```

Comment:
we just created a viewmodel for this class, maybe we should use it (not a must tho)